Controller/ConnectController.php

```diff
@@ -38,8 +39,10 @@
 /**
  * @author Alexander <iam.asm89@gmail.com>
  */
-class ConnectController extends Controller
+class ConnectController extends AbstractController
 {
+    use ControllerTrait;
```

Comment:
Maybe I'm missing something, but `AbstractController` already imports the `ControllerTrait`:

https://github.com/symfony/symfony/blob/4.2/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L39